atomicapp/applogging.py

```diff
@@ -100,6 +101,8 @@ def setup_logging(verbose=None, quiet=None, logtype=None):
         The 'color' level prints out normal log msgs (no cockpit) with color
         """
 
+        # setting the root directory path of code base
+        customOutputFormatter.path = '/'.join(__file__.split('/')[:-1])
```

Comment:
I don't really like setting it here. I'd rather have it set as a constant named something like `atomicapproot` as a class variable maybe on line 33 of this file?